File: SampleSetup/Protocol/Body/ApiReturnInterface.cs
using System.Collections.Generic;

namespace SampleProject.Protocol.Body
{
    [UsingRef(nameof(AbstractReturn))]
    public interface ApiReturnInterface
    {
        AbstractReturn Payload();
        string? Meta();
        string? Fill(string? p1, List<int> p2, List<double> p3, int? p4);
    }
}

File: SampleSetup/Output/PHP_8/Body/ApiReturnInterface.php
<?php
namespace SampleProject\Protocol\Body;

interface ApiReturnInterface {
    function Payload():         \SampleProject\Protocol\Body\AbstractReturn;
    function Meta(): string;
    /**
     * @param int[] $p2
     * @param float[] $p3
     */
    function Fill(?string $p1, $p2, $p3, ?int $p4): string;
}
